Dockerfiles/Dockerfile.dela

```diff
@@ -2,12 +2,16 @@ FROM golang:1.20.6-bookworm
 
 RUN apt-get update && apt-get install git
 RUN git clone https://github.com/dedis/dela.git
-RUN git clone https://github.com/dedis/d-voting.git
+WORKDIR /go/dela
+# make sure we're using the same head as d-voting
+RUN git checkout fix-bbolt
```

Comment:
`git clone ... -b fix-bbolt`

Reply:
why? this creates a new branch, I want the head to be on the existing one

Reply:
https://git-scm.com/docs/git-clone#Documentation/git-clone.txt--bltnamegt

Reply:
ah, sorry I misread I didn't see the `clone`, I'll change that